third_party/freedom-typings/freedom-common.d.ts

```diff
@@ -134,6 +134,22 @@ declare module freedom {
     // TODO: explore how to use FreedomModuleFactoryManager.
     [moduleName:string] : FreedomModuleFactoryManager<any>;
   }
+
+  // This generic interface represents any freedom method. Its purpose is to extend
+  // the basic definition to include the reckless call method, which does not
+  // produce a reply message.
+  interface Method1<T,R> {
```

Comment:
Yeah, `Method` is not a great name ... but I'm not sure `Reckless` is right either, because _every_ method supports this pattern.